src/utils/getFilteredTodos.tsx

```diff
@@ -0,0 +1,26 @@
+import { Todo } from '../types/Todo';
+import { TodoFilter } from '../types/TodoFilter';
+
+export const getFilteredTodos = (
+  todos: Todo[],
+  selectedFilter: string,
+) => {
+  let preparedTodos = [...todos];
+
+  if (selectedFilter !== TodoFilter.All) {
+    preparedTodos = preparedTodos.filter(({ completed }) => {
+      switch (selectedFilter) {
```

Comment:
```suggestion
  if (selectedFilter !== TodoFilter.All) {
    return todos.filter(({ completed }) => {
      switch (selectedFilter) {
```
`filter` returns new array so you don't need create a copy

Reply:
😅ooopsss